### ISSUE TITLE
kann_layer_layernorm = batch normalization?

### ISSUE THREAD

**Issue by abylouw** [attractivechaos/kann#18]
Hi,

Is kann_layer_layernorm the same as batch normalization?

Thank you

**Comment by attractivechaos** [comment]
No. Layernorm is different from batchnorm. KANN unfortunately doesn't have batchnorm.
